### ISSUE TITLE
how the ByteTrack works while in low frame rate 8 frames/s e.g.

### ISSUE THREAD

**Issue by leoluopy** [FoundationVision/ByteTrack#64]
hi , @ifzhang 
     thanks for great works , and have u tried , how it works in low frame rate situation , 8 frames/s for example . i tried it with my own detector in 8 frames/s , but it didn't work well . it seems while in low frame rate , trackers with embedding still works better , do you have any idea ?

**Comment by ifzhang** [comment]
Yes, ByteTrack only uses Kalman Filter and it does not work well in low frame rate situation. You can combine BYTE with Re-ID features (tutorials/fairmot) in low frame rate videos.

**Comment by AlexTindeche** [comment]
Hey @ifzhang,
Can you provide a clarification? I do not really understand what I should be doing to make BYTETrack work with Re-ID. Currently I am using the yolox BYTETyracker as presented below:
```python
from yolox.tracker.byte_tracker import BYTETracker, STrack
from onemetric.cv.utils.iou import box_iou_batch
from dataclasses import dataclass


@dataclass(frozen=True)
class BYTETrackerArgs:
    track_thresh: float = 0.25
    track_buffer: int = 30
    match_thresh: float = 0.8
    aspect_ratio_thresh: float = 3.0
    min_box_area: float = 1.0
    mot20: bool = False
```
How can I make it work with the Re-ID from FairMOT?

Thank you

**Comment by masip85** [comment]
Why kalman filter cannot be adjusted for low frame rate scenario?
